scripts/apps/authoring/track-changes/suggestions.js

```diff
@@ -0,0 +1,100 @@
+import {
+    editor3DataKeys,
+    getCustomDataFromEditor
+} from 'core/editor3/helpers/editor3CustomData';
+
+function getAllUserIdsFromSuggestions(suggestions) {
+    const users = [];
+
+    suggestions.forEach(({resolutionInfo, suggestionInfo}) => {
+        users.push(resolutionInfo.resolverUserId);
+        users.push(suggestionInfo.author);
+    });
+
+    return users.filter((value, index, self) => self.indexOf(value) === index);
+}
+
+function getUsersAsQueryArray(suggestions) {
+    return getAllUserIdsFromSuggestions(suggestions)
+        .map((key) => ({_id: key}));
+}
+
+function convertUsersArrayToObject(users) {
+    const usersObj = {};
+
+    users.forEach((user) => {
+        usersObj[user._id] = user;
+    });
+
+    return usersObj;
+}
+
+function getUsersFromAPI(api, query) {
+    return api.query('users', {where: JSON.stringify(query)});
```

Comment:
instead of using api you could use [userList](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/users/services/UserListService.js#L15) service which has all users cached

Reply:
In that service, how can I get the users of several `_id` at the same time?

Reply:
you can use `getAll` and do the filtering here